runtime/all/all_with_podman.go

```diff
@@ -0,0 +1,11 @@
+//go:build linux && podman
+// +build linux,podman
+
+package all
+
+import (
+	_ "github.com/srl-labs/containerlab/runtime/containerd"
+	_ "github.com/srl-labs/containerlab/runtime/docker"
+	_ "github.com/srl-labs/containerlab/runtime/ignite"
+	_ "github.com/srl-labs/containerlab/runtime/podman"
+)
```

Comment:
do we still need build tags to have a build with podman, considering that you managed to exclude cgo dependency?

Reply:
In principle - we don't; but the build process would the require presence of the other tags i mentioned earlier:
`exclude_graphdriver_btrfs btrfs_noversion exclude_graphdriver_devicemapper exclude_graphdriver_overlay containers_image_openpgp`

In the current form this is a safety first approach: in case of no tags `go build` will succeed without podman (based on runtime/all/all.go); otherwise we expect to see all the tags (I even contemplated to include them all).

It's a little bit ugly either way. 😞
I suspect this approach might be more straightforward for external contributors..

Reply:
yes, that is what I thought, but once this PR is done, I think it would be best to include podman support to the GA build of containerlab (ofc we will include those tags to a regular build pipeline delivered by goreleaser)

That way it would be possible to let users with podman to play with it, without fetching separate binary. At this point, the binary increase that podman will incur won't matter much I guess.

Reply:
> I think it would be best to include podman support to the GA build of containerlab

I mean, that is the plan;
the question is whether we want to keep an option to build clab without podman support for other contributors?